tests/spread/core24/package-repositories/test-key-conflict/snapcraft.yaml

```diff
@@ -0,0 +1,31 @@
+name: test-key-conflict
+version: '1.0'
+summary: test package repos with keys already present on the system
```

Comment:
Like the well-designed tests above, this is a well-written example that makes it really clear. Thanks!